helpers.py

```diff
@@ -45,26 +45,19 @@ def get_nested(meter_data, path):
     return value
 
 
-def get_default_template_config(config: dict, template_number: int, name: str, defaultvalue: any = None) -> any:
-    '''check if config value exist in TEMPLATE section, otherwise return defaultvalue'''
-    if name in config[f"TEMPLATE{template_number}"]:
-        return config[f"TEMPLATE{template_number}"][name]
-
-    return defaultvalue
-
-
-def try_get_value(value: str, expected_type: [str, int, float, bool],
-                  new_value: [None, str, int, float, bool]) -> [None, str, int, float, bool]:
+def parse_to_expected_type(value: str, expected_type: [str, int, float, bool],
```

Comment:
why "parse_..." not "convert..." ?

Reply:
Because it just parses a value into the given type. But I think it's pretty similar and so I renamed it to convert.